projects/plugins/jetpack/extensions/blocks/anchor-fm/templates/index.js

```diff
@@ -76,22 +76,31 @@ function podcastSection( { episodeTrack, feedUrl, coverImage } ) {
 }
 
 function podcastSummarySection( { episodeTrack } ) {
-	const summaryBlocks = pasteHandler( { HTML: episodeTrack.description_html, mode: 'BLOCKS' } );
-	return [
-		'core/group',
-		{},
+	const sectionBlocks = [
 		[
-			[
-				'core/heading',
-				{
-					level: 3,
-					content: 'Summary',
-					placeholder: __( 'Podcast episode title', 'jetpack' ),
-				},
-			],
-			...summaryBlocks,
+			'core/heading',
+			{
+				level: 3,
+				content: 'Summary',
+				placeholder: __( 'Podcast episode title', 'jetpack' ),
+			},
 		],
 	];
+
+	const summaryBlocks = pasteHandler( { HTML: episodeTrack.description_html, mode: 'BLOCKS' } );
+
+	if ( summaryBlocks.length ) {
```

Comment:
I guess we're sure that it will be always an array, despite it'll be empty or not.

Reply:
That's what I found when testing--passing an empty string as `HTML` results in an empty array of blocks.